src/core/components/AppTransactionFee.vue

```diff
@@ -0,0 +1,270 @@
+<template>
```

Comment:
since this is only used at one place (swap) i think ..it can go under that folder instead of this one, this folder will eventually be removed

Reply:
This is fine since it'll be used on Send as well.

Reply:
should it be moved to Core components then?

Reply:
Actually you're right, this is only used on Swap it looks like. It should just be placed within the swap folder

Reply:
We use this in send page too.